### PR TITLE
default generate deliver file

### PR DESCRIPTION
**Pull request by ohwutup**

### CODE REVIEW

**Comment by KrauseFx** [comment]
Why did you change the methods? I think the method names are kind of unclear, we can reduce the number of methods here

File: lib/deliver/setup.rb [hunk 1]
```diff
@@ -5,13 +5,13 @@ def run(options)
       create_based_on_identifier(containing, options)
     end
 
-    # This will download all the app metadata and store its data into JSON files
-    # @param deliver_path (String) The directory in which the Deliverfile should be created
-    # @param identifier (String) The app identifier we want to create Deliverfile based on
-    # @param project_name (String) The default name of the project, which is used in the generated Deliverfile
```
Comment (ohwutup) [inline lib/deliver/setup.rb]: only pass app id instead of options

File: lib/deliver/setup.rb [hunk 2]
```diff
@@ -5,13 +5,13 @@ def run(options)
...
     def create_based_on_identifier(deliver_path, options)
       file_path = File.join(deliver_path, 'Deliverfile')
       data = generate_deliver_file(deliver_path, options)
+      setup_deliver(file_path, data, deliver_path, options)
```
Comment (ohwutup) [inline lib/deliver/setup.rb]: move this up to run

File: lib/deliver/setup.rb [hunk 3]
```diff
@@ -34,7 +34,7 @@ def generate_deliver_file(deliver_path, options)
       deliver = File.read("#{gem_path}/lib/assets/DeliverfileDefault")
       deliver.gsub!("[[APP_IDENTIFIER]]", options[:app].bundle_id)
       deliver.gsub!("[[USERNAME]]", Spaceship::Tunes.client.user)
-
+      deliver += "\napple_id ENV['PRODUCE_APPLE_ID']" if ENV['PRODUCE_APPLE_ID']
```
Comment (ohwutup) [inline lib/deliver/setup.rb]: get rid of this

File: lib/deliver/setup.rb (hunk 1 above)
Comment (KrauseFx) [inline lib/deliver/setup.rb]: `app_identifier`

File: lib/deliver/setup.rb (hunk 2 above)
Comment (KrauseFx) [inline lib/deliver/setup.rb]: Move the whole implementation of `create_based_on_identifier` up to `run`

**Comment by KrauseFx** [comment]
:rocket:

**Pull request merged**